proofOfConcept/ColorPickerAlpha/ColorPickerAlpha/MainWindow.xaml.cs

```diff
@@ -32,24 +33,67 @@ public MainWindow()
                         G_val.Text = curColor.G.ToString();
                         B_val.Text = curColor.B.ToString();
 
+                        HEXValue.Text = rgbToHEX(curColor.ToString());
+
                     }));
 
                     Thread.Sleep(100);
                 }
             }).Start();
         }
 
+        private void toggle_rgb(object sender, RoutedEventArgs e)
+        {
+            rgbState = !rgbState;
+            if (rgbState)
+            {
+                R_val.Visibility = Visibility.Visible;
+                RLabel.Visibility = Visibility.Visible;
+                G_val.Visibility = Visibility.Visible;
+                GLabel.Visibility = Visibility.Visible;
+                B_val.Visibility = Visibility.Visible;
+                BLabel.Visibility = Visibility.Visible;
+                HEXValue.Visibility = Visibility.Hidden;
+                HEXLabel.Visibility = Visibility.Hidden;
+            }
+            else
+            {
+                R_val.Visibility = Visibility.Hidden;
+                RLabel.Visibility = Visibility.Hidden;
+                G_val.Visibility = Visibility.Hidden;
+                GLabel.Visibility = Visibility.Hidden;
+                B_val.Visibility = Visibility.Hidden;
+                BLabel.Visibility = Visibility.Hidden;
+                HEXValue.Visibility = Visibility.Visible;
+                HEXLabel.Visibility = Visibility.Visible;
```

Comment:
Could just do
var visibility = rgbState ? Visibility.Visible : Visibility.Hidden, then
R_val.Visibility = G_val.Visibility = ... = visibility
and
HEXLabel.Visibility = !visibility